modern/src/components/NavBar.js

```diff
@@ -17,12 +16,10 @@ const ReportNavbar = ({ setOpenDrawer, reportTitle }) => (
         <MenuIcon />
       </IconButton>
       <Typography variant="h6" noWrap>
-        {t('reportTitle')}
-        {' '}
-        {` / ${reportTitle}`}
+        {title}
```

Comment:
I think it would be nice to use proper breadcrumbs here.

https://material-ui.com/components/breadcrumbs/

Reply:
I don't think breadcrumbs will be ideal here. Even though the idea is similar to breadcrumbs in regards to letting the user know where they are, the experience of using it is intended to be different. 

Example: if we change this into breadcrumbs, clicking the `Options` or `Report` link will take the user to the default report or option. This will be `Reports / Routes` or `Options / Account`. Unless we create a list or separate Reports and Options pages which lists the available pages. 

We end up duplicating the functionality of the humburger menu if we create separate pages or list